inc/functions.inc

```diff
@@ -10252,6 +10252,29 @@ if (!function_exists("livesearch_sarjanumerohaku")) {
   }
 }
 
+if (!function_exists("hae_kayttajan_toimipistetunnukset")) {
+  function hae_kayttajan_toimipistetunnukset() {
+    global $kukarow;
+  
+    $asiakastunnukset = '';
+    if (!empty($kukarow['oletus_asiakas'])) {
+      $asiakastunnukset = $kukarow['oletus_asiakas'];
+    }
+
+    $query = "SELECT group_concat(asiakas.tunnus) toimipistetunnukset
+              FROM customers_users 
+              JOIN asiakas ON (customers_users.customer_id = asiakas.tunnus) 
+              WHERE user_id = '{$kukarow['tunnus']}'";
+    $result = pupe_query($query);
+    if (mysql_affected_rows() == 1) {
```

Comment:
@mikamyllynen 

    mysql_affected_rows () - Get the number of affected rows by the last INSERT, UPDATE, REPLACE or DELETE query.

Tämä ei taida antaa toivottua tulosta SELECTeissä?